src/StatusQ/Controls/StatusFlatButton.qml

```diff
@@ -14,7 +14,7 @@ StatusBaseButton {
                                                      : Theme.palette.dangerColor1
     disabledTextColor: Theme.palette.baseColor1
 
-    border.color: type === StatusBaseButton.Type.Normal || hovered ? "transparent"
+    borderColor: type === StatusBaseButton.Type.Normal || hovered ? "transparent"
```

Comment:
I assume this is a result of moving to `Button` type in `StatusBaseButton` as well?

Reply:
removed